fix: add missing migration from player death column

dudantas opened the pull request:
Related to pr: #3517 (e2952832789ce814ce98db6d1d5548d1f977b779)

dudantas committed:
fix: add missing migration from player death column

QuebradaZN commented:
Participating monsters were Morshabaal and Mawhawk
On the Kill List, if you look closely, only Morshabaal and participant Morshabaal appear
![image](https://github.com/user-attachments/assets/47c40e3e-835e-4322-93a7-862ecb647108)

otmexa commented:
Is DEFAULT '[]' necessary? Because that causes an error when exporting and trying to import the database.

dudantas committed:
Update schema.sql

Pablodotnet commented:
I made an issue about this bug

Pablodotnet commented:
https://github.com/opentibiabr/canary/issues/3612

dudantas committed:
Update 50.lua

dudantas committed:
Merge branch 'main' into dudantas/add-missing-migration

dudantas committed:
Add participants column to player_deaths table

Introduces a database migration that adds a 'participants' TEXT column to the 'player_deaths' table, with a default empty string. This supports tracking participants in player deaths.

github-actions[bot] committed:
Lua code format - (Stylua)

majestyotbr committed:
fix: migration and schema
